core/api/service/state/requests/get_storage.hpp

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_API_REQUEST_GET_STORAGE
+#define KAGOME_API_REQUEST_GET_STORAGE
+
+#include <jsonrpc-lean/request.h>
+
+#include <boost/optional.hpp>
+#include <common/buffer.hpp>
+#include <outcome/outcome.hpp>
+#include <primitives/block_id.hpp>
+
+#include "api/service/state/state_api.hpp"
+
+namespace kagome::api::state::request {
+
+  class GetStorage final {
+   public:
+    GetStorage(std::shared_ptr<StateApi> api) : api_(std::move(api)){};
```

Comment:
Won't it be better to declare it explicit? And, perhaps, define in the source file just to be consistent.

Reply:
Agree